invenio/celery/__init__.py

```diff
@@ -159,3 +175,7 @@ def autodiscover(self):
     'invenio',
     loader=InvenioLoader,
 )
+celery.setup_security(
```

Comment:
Can't this be done in the InvenioLoader init (first parameter is the Celery app)?